template/src/api/routes/$$channel$$.js

```diff
@@ -1,79 +1,126 @@
-const Router = require('hermesjs/lib/router');
-const {validateMessage} = require('../../lib/message-validator');
-const router = new Router();
-const {{ channelName | camelCase }}Handler = require('../handlers/{{ channelName | convertToFilename }}');
-module.exports = router;
+import { File } from "@asyncapi/generator-react-sdk";
```

Comment:
why with react you still want to use `file templates` (`$$channel$$.js`) instead new way? look at https://github.com/asyncapi/generator/pull/1117#issuecomment-2006369580 for more details